integration_tests/pages/workAndSkillsPage.ts

```diff
@@ -82,27 +82,17 @@ export default class WorkAndSkillsPage extends Page {
   ES_skillLevelOne = (): PageElement => cy.get('.govuk-grid-column-two-thirds > p')
 
   // Goals card
-  GoalsCard = (): PageElement => cy.get('#goals')
+  GoalsInfo = (): PageElement => cy.get('[data-qa="goals-info-text"]')
 
-  GoalsHeader = (): PageElement => cy.get('#goals > [data-qa="summary-header"] > h2')
+  PlpGoalsSummary = (): PageElement => cy.getDataQa('plp-goals')
 
-  GoalsInfo = (): PageElement => cy.get('#goals > .hmpps-summary-card__body > p')
+  Vc2GoalsSummary = (): PageElement => cy.getDataQa('vc2-goals')
 
-  GoalsEmploymentLabel = (): PageElement => cy.get('#goals > .hmpps-summary-card__body > :nth-child(2)')
+  PlpVc2GoalsSummary = (): PageElement => cy.getDataQa('plp-vc2-goals')
 
-  GoalsEmploymentText = (): PageElement => cy.get('#goals > .hmpps-summary-card__body > :nth-child(3)')
+  NoGoalsSummary = (): PageElement => cy.getDataQa('no-goals')
```

Comment:
I'm not overly keen on the naming starting with a capital letter, but I can see thats what everything else has in this file, so you were right to follow the existing convention 👍